tee-worker/litentry/primitives/src/lib.rs

```diff
@@ -67,6 +68,8 @@ pub enum LitentryMultiSignature {
 	Ecdsa(ecdsa::Signature),
 	/// An ECDSA/keccak256 signature. An Ethereum signature. hash message with keccak256
 	Ethereum(EthereumSignature),
+	/// Same as the above, but the payload bytes are hex-encoded and prepended with a readable prefix
+	EthereumPrettified(EthereumSignature),
```

Comment:
We could have another varaint for EIP-191 too 😂

Reply:
Yeah, thought about that :joy: Was trying to keep the scope narrow here for the sake of the release; happy to follow up on it later tho.

Reply:
That's true - but let's get it over the line and launch the campaign first 😂

Reply:
Might not be that easy as we will get nested variants Prettified EIP 🤔  ?
Approved as this is not blocking for me.